typescript-timers/interval.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const $countdownDisplay = document.querySelector('.countdown-display');
+if (!$countdownDisplay) throw new Error('$countdownDisplay query failed');
+let count = 4;
+const intervalId = setInterval(function () {
+  count--;
+  if (count > 0) {
+    $countdownDisplay.textContent = String(count);
```

Comment:
Ideally you want to use the `toString` method to convert one type to a string rather than using the `String` constructor function. Essentially when create the string with the constructor function, it will still call the `toString` method at the end of it so you should skip the steps of creating new objects to convert to a string.